validation/dam_break_2d/validation_dam_break_2d.jl

```diff
@@ -95,22 +95,26 @@ viscosity_edac = ViscosityAdami(nu=alpha * smoothing_length * sound_speed / 8)
 fluid_system_edac = EntropicallyDampedSPHSystem(tank_edac.fluid, smoothing_kernel,
                                                 smoothing_length,
                                                 sound_speed, viscosity=viscosity_edac,
+                                                density_calculator=SummationDensity(),
```

Comment:
I thought `ContinuityDensity` is supposed to be better, @LasNikas?

Reply:
Yes, `density_calculator=ContinuityDensity()` and `pressure_acceleration=nothing`

Reply:
As far as I understand the result is not better just faster. But I just didn't want to update the reference files.

Reply:
`SummationDensity` is more exact for closed surface and for long sim times. In the dam break case, there is almost no difference when using `ContinuityDensity` and `SummationDensity` except the  very shorter runtime. 

In steady free surface problems, `ContinuityDensity` gives better results. 

However, I didn't find something in the literature about that, since the edac method for SPH is very young. 
I could produce similar results for `ContinuityDensity` and `SummationDensity` for a closed surface problem when using a different laplace discretization formulation (see #392 ).

Reply:
I have updated the reference files. The result is either the same or slightly worse but not better.